Curtains/Curtains.Domain/Models/BaseEntity.cs

```diff
@@ -16,12 +16,8 @@ public class BaseEntity
         /// </summary>
         public int Id { get; set; }
         /// <summary>
-        ///  This property represents title of entities.
-        /// </summary>
-        public string Title { get; set; } = "";
```

Comment:
Why did you remove Title property?

Reply:
cuz it is not used in all models and conflicts with the structure of database

Reply:
Egor said to add Title and Description to BaseEntity class in previous pull request

Reply:
fixed :)